actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+*    Add support for passing array of records to `fresh_when` and `stale?` via
+     `Enumerable#maximum`.
+
+     ```ruby
+     def index
+       @payments = current_user.payments.limit(20).to_a
+
+       fresh_when @payments
+     end
+     ```
+
+     It is the equivalent of writing:
+
+     ```ruby
+     def index
+       @payments = current_user.payments.limit(20).to_a
+
+       fresh_when @payments, last_modified: @subscriptions.maximum(:updated_at)
```

Comment:
Don't think these are comparable, when there's a `@payments` variable and then a `@subscriptions` variable?

Reply:
Whoops typo `@subscriptions` should be `@payments`